src/main/java/nl/moreniekmeijer/backendspringboottechiteasycontroller/services/WallBracketService.java

```diff
@@ -0,0 +1,67 @@
+package nl.moreniekmeijer.backendspringboottechiteasycontroller.services;
+
+import nl.moreniekmeijer.backendspringboottechiteasycontroller.dtos.WallBracketInputDto;
+import nl.moreniekmeijer.backendspringboottechiteasycontroller.dtos.WallBracketResponseDto;
+import nl.moreniekmeijer.backendspringboottechiteasycontroller.exceptions.RecordNotFoundException;
+import nl.moreniekmeijer.backendspringboottechiteasycontroller.mappers.WallBracketMapper;
+import nl.moreniekmeijer.backendspringboottechiteasycontroller.models.WallBracket;
+import nl.moreniekmeijer.backendspringboottechiteasycontroller.repositories.WallBracketRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class WallBracketService {
+    private final WallBracketRepository wallBracketRepository;
+
+    public WallBracketService(WallBracketRepository wallBracketRepository) {
+        this.wallBracketRepository = wallBracketRepository;
+    }
+
+    public WallBracketResponseDto addWallBracket(WallBracketInputDto wallBracketInputDto) {
+        WallBracket createdWallBracket = wallBracketRepository.save(WallBracketMapper.toEntity(wallBracketInputDto));
+        return WallBracketMapper.toResponseDto(createdWallBracket);
+    }
+
+    public List<WallBracketResponseDto> getAllWallBrackets() {
+        List<WallBracket> foundWallBrackets = wallBracketRepository.findAll();
+        return foundWallBrackets.stream()
+                .map(WallBracketMapper::toResponseDto)
+                .toList();
+    }
+
+    public WallBracketResponseDto getWallBracketById(Long id) {
+        if (id < 0) {
```

Comment:
Deze if is in theorie niet nodig. je kijkt namelijk of er uberhaupt een wallbracket is met het ingevoerde ID daarna als je niks terug krijgt bestaat hij niet en is hij dus out of bounds :)